WSD ID 7962, 9104 and 11191. Pull request for the quarterly updates.

yutakats opened the pull request:
Pull request for the quarterly updates to go with the product version upgrades & recent OCI UI changes. 

# WMS IDs:
1. WSD ID: 7962, Setup OpenTracing for Microservices with Helidon using Oracle APM
2. WSD ID: 9104, Monitor Traces for WebLogic Server Deployed on Kubernetes using Oracle APM
3. WSD ID 11191, Automatically instrument Java microservices on Kubernetes using Oracle APM

# General requirements - verified
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
5. Make sure you enter the help email link in your manifest.json
6. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x ] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

arabellayao reviewed (changes requested):
Hi! Please fix the following issues of your PR:
1. Your branch branch is out-of-date with the oracle-livelabs main branch. Please merge the latest changes from main into this branch.
2. It seems like you are updating files outside of the 3 workshops specified. Please add the WMS IDs of all workshops that have been updated.
3. Images’ names should be descriptive instead of step numbers

yutakats commented:
Hi Arabella, 

1. I have added missing WMS ID for the 4th workshop. Sorry I missed one change i had in this workshop. 
2. I merged upstream/main and pushed origin again as following the steps in the step-by-step lab 4 task 1.  Please let me know if anything still missing to do. Sorry i should have read the doc first. 

3. As for the image files have Step-Ids+descriptive names... as in 3-1-2-domain.png. Which shows for this example, a screenshot of the domain page. Is it not good enough..? 
![image](https://user-images.githubusercontent.com/91303359/208270662-4af28014-92ef-4066-a220-db59a301efbc.png)

Thank you and have a great rest of the weekend. 

Yutaka

arabellayao reviewed (changes requested):
Hi! Please fix the following issues of your PR:
1. Can you add the WMS ID of the oci-apm-react-helidon-atp?
2. For the cloud shell icon,
<img width="345" alt="Screen Shot 2022-12-19 at 11 51 15 AM" src="https://user-images.githubusercontent.com/48001060/208508368-442c7818-a9fe-40b7-ab5f-9bec257a3653.png"> we suggest you use this common image: https://oracle-livelabs.github.io/common/images/console/cloud-shell.png - this way, if there is any change to the OCI console, our LiveLabs Team can update the image in your workshop at the same time
3. As for image filenames, you can include both the step number and the description. But if you have multiple images with different step number and the same description, please change the description to be different (unique) as well. When you add/remove a step, you don't need to change the filenames again.
4. Please make sure image names are all lower case:
<img width="347" alt="Screen Shot 2022-12-19 at 11 44 59 AM" src="https://user-images.githubusercontent.com/48001060/208507348-d1f8633b-b19f-4aee-95fc-12d140cee968.png">
Thanks

yutakats commented:
1. I added the WMS ID 7969 for the Trace a "Native Cloud App..." workshop. But my apology. I forgot the fact that I added a change in the lab folders, in that workshop, by adding index numbers to them. Which ended up replacing all files within the folders. I reverted back all the files in this project to their original state. As you recall, we later decided to disable this workshop as we spoke in slack - so we do not need that changes anymore. 

2 & 3. I have changed the file names for cloud shells to "1-1-menu-cloudshell.png" and "1-2-cloudshell-check-versions.png" to distinguish the images from the names. I would like to keep the index numbers for sorting reasons. It will be very hard to maintain from dev perspectives without them. 

4. The tree in the screenshot is from the WMS ID 7969 and all changes are reverted back to the original state. If this is not good enough I can remove the entire workshop folder. 

I do notice this now that I had to make changes to the existing image files too, but can I do that in the next round of effort? I have thousands of the images and that would be a major update to the workshops.

yutakats commented:
Hi Arabella.

I have replaced the image of the menu item of "CloudShell" with the suggested image. 

Thank you,

Yutaka

arabellayao reviewed (approved):
Thank you!

yutakats commented:
Correction to my previous change - Noticed the suggestion was to remove the local file and link to the commonly-used file at the shared location. Done the change.
